### PR TITLE
Math

### PR DESCRIPTION
**Pull request by veronaten**
Section Mathematic functions, task 13, 14.

### CODE REVIEW

File: scripts/math.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+function runArithmetic(array) {
```
Comment (magicsd) [inline scripts/math.js]: What does this naming mean?

File: scripts/math.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  if (!Array.isArray(array) || array.length < 2) {
+    alert('Enter minimum 2 numbers separated by space, please');
+  }
+
+  return array[i] / 2;
```
Comment (magicsd) [inline scripts/math.js]: `i` is undefined here

File: scripts/math.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+function findMean() {
+  const userNumbers = prompt('Enter some integer numbers separated by spaces', '12 15 20 25 59 79') || '0';
+
+  const array = userNumbers.split(' ');
+
+  let result = 0;
+
+  for (i = 0; i < array.length; i++) {
+    result = result + runArithmetic(array);
+  }
```
Comment (magicsd) [inline scripts/math.js]: What does this expression do? Are you sure it works as you expect it to work?

File: scripts/math.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+  alert(`The arithmetic mean of your array is ${result}.`);
+}
+
+function findFactorial() {
+  let inputNumber = Number(prompt('Enter a number', '5') || 0);
```
Comment (magicsd) [inline scripts/math.js]:
```suggestion
  const inputNumber = Number(prompt('Enter a number', '5') || 0);
```

File: scripts/math.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  if (Number.isNaN(inputNumber)) {
+    alert('Enter a number, please.');
+
+    return;
+  }
+
+  let result = 1;
+
+  for (let i = 1; i < inputNumber; i++) {
+    result = result * i;
```
Comment (magicsd) [inline scripts/math.js]:
```suggestion
    result *= i;
```

File: scripts/math.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  }
+  
+  let factorial = result * inputNumber;
```
Comment (magicsd) [inline scripts/math.js]:
```suggestion
  const factorial *= inputNumber;
```

File: scripts/math.js (hunk 1 above)
Comment (veronaten) [reply scripts/math.js]: It means, that we are running arithmetic mean. But I deleted 'mean', maked name  shorter.

**Pull request merged**